README.md

```diff
@@ -11,10 +11,16 @@ AGRC manages the additions, replacements, and deletions of SGID items in the bro
 
 1. When an application, dataset, or both needs to be deleted or moved to a different long-term storage, [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
 1. Choose the `Deprecate SGID Dataset or Web Application` template from the buttons
-1. Tag the three triagers (one from each team) in the triage section
-1. The triagers will choose which items need to be handled and tag the appropriate people to handle them
-   - Items that don't need to be handled should be struck through (wrap the line with `~`'s) instead of being deleted
-1. The people assigned to different issues will check the boxes as they are completed and comment on the issue to notify the rest of the group.
+1. Tag the three triagers (one from each team) in the triage section:
+   - [ ] Data Team Triage (@gregbunce)
```

Comment:
**thought**: I wonder if it would be more helpful to give examples as code:

```suggestion
   - `[ ] Data Team Triage (@gregbunce)`
```

Reply:
I see what you're saying, that would help the author/triagers. I was kinda thinking about the users assigned to fix things, wondering if the rendered markdown was more applicable to what they will see.